assets/agenda/actions.ts

```diff
@@ -590,15 +590,15 @@ export function setAndUpdateNewItems(data: any) {
             return Promise.resolve();
         }
 
-        dispatch(updateItem(item));
+        const {itemsById} = state;
+        const prevItem = itemsById[item['_id']];
 
-        // Fetch related planning items if the updated item has planning_items
-        if (item.item_type === 'event' && item.planning_items && item.planning_items.length > 0) {
-            item.planning_items.forEach((plan: IAgendaItem) => {
-                dispatch(fetchItem(plan._id));
```

Comment:
I wonder if we're still fine in this case when there is the planning items updated if that wouldn't change the coverages

Reply:
I checked that part it works fine, If we fetch all the items, they also update its related planning items
so we don't need to fetch its related planing item one by one.

Reply:
previously, we implemented this for coverages updates

Reply:
ok

Reply:
my blunders, we require this code; I re-checked and noticed one case is failing.